packages/imperative/__tests__/Imperative.test.ts

```diff
@@ -233,6 +234,35 @@ describe("Imperative", () => {
 
                 expect(ConfigManagementFacility.instance.init).toHaveBeenCalledTimes(1);
             });
+
+            it("should surface failures if daemonMode is specified", async () => {
```

Comment:
Should this say `if daemonMode is not specified`?

Reply:
Yeah...

Reply:
Fixed.